rdkit/Chem/Draw/__init__.py

```diff
@@ -529,3 +531,165 @@ def MolToQPixmap(mol, size=(300, 300), kekulize=True, wedgeBonds=True, fitImage=
   drawer.AddMol(mol, **kwargs)
   canvas.flush()
   return canvas.pixmap
+
+
+def DrawMorganBit(mol, bitId, bitInfo, whichExample=0, **kwargs):
+  atomId, radius = bitInfo[bitId][whichExample]
+  return DrawMorganEnv(mol, atomId, radius, **kwargs)
+
+
+# adapted from the function drawFPBits._drawFPBit() from the CheTo package
+# original author Nadine Schneider
+def DrawMorganEnv(mol, atomId, radius, molSize=(150, 150), baseRad=0.3, svg=True,
```

Comment:
Awesome.  I might try and port this to C++ to make it accessible to java/c#, I think it would be very useful there and this code looks quite amenable to porting.